src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -8,35 +8,30 @@ import {
   Pagination,
   PaginationVariant,
   Spinner,
-  TextInput,
 } from '@patternfly/react-core';
-import {
-  ActionsColumn,
-  IAction,
-  TableComposable,
-  Tbody,
-  Td,
-  Th,
-  Thead,
-  Tr,
-} from '@patternfly/react-table';
+import { ActionsColumn, IAction, TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
 import { global_BackgroundColor_100 } from '@patternfly/react-tokens';
-import { useState } from 'react';
+import React, { useContext, useState } from 'react';
 import { createUseStyles } from 'react-jss';
 import { useQueryClient } from 'react-query';
 import { ContentItem } from '../../services/Content/ContentApi';
 
-import {
-  useContentListQuery,
-  useDeleteContentItemMutate,
-} from '../../services/Content/ContentQueries';
+import { useContentListQuery, useDeleteContentItemMutate } from '../../services/Content/ContentQueries';
+import ContentListFilters from './ContentListFilters';
+import ContentListChips from './ContentListChips';
+import { ContentListContext } from './ContentListContext';
 
 const useStyles = createUseStyles({
   actionContainer: {
     backgroundColor: global_BackgroundColor_100.value,
     justifyContent: 'space-between',
     padding: '16px 24px',
   },
+  chipsContainer: {
+    backgroundColor: global_BackgroundColor_100.value,
+    justifyContent: 'justifyContentFlexStart',
```

Comment:
does this work? I would think that the correct format would be flex-start here, as "justifyContentFlexStart" is specific to patternfly.

Reply:
i changed it to 'flex-start' and it still looks okay, so I'll leave it like that. I don't remember where "justifyContentFlexStart" came from.